README.md

```diff
@@ -2,42 +2,64 @@
 
 This repo contains deployment scripts executed during project builds on [Travis CI](https://travis-ci.org) and [Gitlab CI](https://about.gitlab.com/features/gitlab-ci-cd/)
 
-The scripts should work across a number of repos that include a `Dockerfile` and a `docker-compose.production.yml` file. They are intended to be included as a [Git submodule](https://git-scm.com/docs/git-submodule) within the code repo.
+The scripts should work across a number of repos that include a `Dockerfile` and a `docker-compose.yml` file within the `/rancher-config/` directory. They are intended to be included as a [Git submodule](https://git-scm.com/docs/git-submodule) within the code repo.
 
-Including this repo within your own (as detailed below) will:
+Including this repo within your re (as detailed below) will:
```

Comment:
not sure why this text changed, should it say 'Including this repo within your own repo'